### DISCUSSION TITLE
Universal Wallet and WebId

### DISCUSSION THREAD

**Discussion by qertis** [walt-id/.github#54] [Ideas / Feature Requests]
I can offer an idea for integrating the [WebID](https://solid.github.io/webid-profile/) and a cryptocurrency wallet within the [Universal Wallet](https://w3c-ccg.github.io/universal-wallet-interop-spec/)

The idea is to authenticate within an application using the cryptocurrency wallet and obtain a standardized JSON-LD document containing all the necessary links to interact with a dApp.

**Comment by mikeplotean** [comment]
Hi, thank you for your input!
Could you please elaborate on what you mean? We already have the web-wallet which has support for verifiable credentials and tokens (it also allows sing-in using a web3 wallet - https://wallet.walt.id).
